acdc/TLACDCExperiment.py

```diff
@@ -313,7 +313,7 @@ def receiver_hook(self, z, hook, verbose=False):
 
         return z
 
-    def add_all_sender_hooks(self, reset=True, cache="first", skip_direct_computation=False, add_all_hooks=False):
+    def add_all_sender_hooks(self, reset=True, cache="first", skip_direct_computation=False, add_all_hooks=False, override=False):
```

Comment:
What does this `override` option do? Would one ever want to set it to `True` ?

Reply:
We do set this to `True` in the `setup_model_hooks` edit from this PR.

The sender hooks had some checks that we're not adding both adding sender and receiver hooks to the same HookPoint. Usually this is a sign something has gone wrong, but in the case where we're adding all the hooks (for test loss calculation) it's not wrong